src/app/store/index.ts

```diff
@@ -33,12 +34,26 @@ const rootReducer = combineReducers({
   analytics: analyticsSlice.reducer,
 });
 
+const migrations = {
+  2: (state: any) => {
+    return {
+      ...state,
+      onboarding: {
+        hasCompletedExploreAppsStep: state.onboarding.stepsStatus['Explore apps'],
+        hasHiddenSuggestedFirstSteps: state.onboarding.hideSteps,
+        hasSkippedFundAccount: false,
+      },
+    };
+  },
+};
+
 const persistConfig = {
   key: 'root',
-  version: 1,
+  version: 2,
   storage,
   serialize: true,
   whitelist: ['keys', 'chains', 'onboarding', 'analytics'],
+  migrate: createMigrate(migrations, { debug: false }),
```

Comment:
Great to see you used migrate, I imagine we'll need to use this eventually.

Though, I do see it as quite some complexity, for a fairly uncomplex state/feature. I think you may have alluded to this previously, but maybe we could just remove `onboading` from the persisted reducer, and great a newly named slice with the refactored state? 

At worst, I think this would leave stale state for existing users, but that wouldn't particularly hurt. If that works, and saves us from doing a migration, I think we should do that.

Reply:
Yeah, I thought about this again too. It is suggested in migration examples as a simpler alternative. I can refactor the migration out. I really just wanted to see it working. Haha.